grass7/gui/wxpython/.flake8

```diff
@@ -0,0 +1,63 @@
+[flake8]
+ignore =
+    E117, # over-indented
+    E121, # continuation line under-indented for hanging indent
+    E122, # continuation line missing indentation or outdented
+    E123, # closing bracket does not match indentation of opening bracket's line
+    E124, # closing bracket does not match visual indentation
+    E125, # continuation line with same indent as next logical line
+    E126, # continuation line over-indented for hanging indent
+    E127, # continuation line over-indented for visual indent
+    E128, # continuation line under-indented for visual indent
+    E131, # continuation line unaligned for hanging indent
+    E202, # whitespace before '}'
+    E203, # whitespace before ':'
+    E211, # whitespace before '('
+    E221, # multiple spaces before operator
+    E222, # multiple spaces after operator
+    E225, # missing whitespace around operator
+    E226, # missing whitespace around arithmetic operator
+    E231, # missing whitespace after ','
+    E241, # multiple spaces after ','
+    E251, # unexpected spaces around keyword / parameter equals
+    E261, # at least two spaces before inline comment
+    E265, # block comment should start with '# '
+    E266, # too many leading '#' for block comment
+    E271, # multiple spaces after keyword
+    E272, # multiple spaces before keyword
+    E301, # expected 1 blank line, found 0
+    E302, # expected 2 blank lines, found 1
+    E303, # too many blank lines (3)
+    E305, # expected 2 blank lines after class or function definition, found 1
+    E306, # expected 1 blank line before a nested definition, found 0
+    E402, # module level import not at top of file
+    E501, # line too long (96 > 79 characters)
+    E722, # do not use bare 'except'
+    E731, # do not assign a lambda expression, use a def
+    E741, # ambiguous variable name 'l'
+    F401, # 'animation.utils.getCpuCount' imported but unused
+    F403, # 'from gmodeler.model import *' used; unable to detect undefined names
+    F405, # '_' may be undefined, or defined from star imports: gmodeler.model
+    F821, # undefined name '_'
+    F841, # local variable 'dc' is assigned to but never used
+    W291, # trailing whitespace
+    W293, # blank line contains whitespace
+    W503, # line break before binary operator
+    W504, # line break after binary operator
+    W605, # invalid escape sequence '\.'
+
+
+max-line-length = 88
+exclude =
+    .git,
+    __pycache__,
+    .env,
+    .venv,
+    env,
+    venv,
+    ENV,
+    env.bak,
+    venv.bak,
+    ctypes,
+    pydispatch,
```

Comment:
As I said, same problem is in both files.

Reply:
fixed